Tools/MiniBrowser/gtk/BrowserWindow.c

```diff
@@ -156,9 +156,7 @@ static void webViewURIChanged(WebKitWebView *webView, GParamSpec *pspec, Browser
 
 static void webViewTitleChanged(WebKitWebView *webView, GParamSpec *pspec, BrowserWindow *window)
 {
-    const char *title = webkit_web_view_get_title(webView);
-    if (!title)
-        title = defaultWindowTitle;
+    const char *title = defaultWindowTitle;
```

Comment:
Oops, sorry I didn't notice earlier. I was about to submit this for merge queue, too.

This shouldn't be removed. Here we set the window title from the web view's title, which has nothing to do with the back/forward list. Remove this change, please.

Reply:
```
    const char *title = webkit_web_view_get_title(webView);
    if (!title)
        title = defaultWindowTitle;
```

> This shouldn't be removed. Here we set the window title from the web view's title, which has nothing to do with the back/forward list. Remove this change, please.

Thanks for catching that — fixed now